mmseg/models/decode_heads/sep_fcn_head.py

```diff
@@ -0,0 +1,52 @@
+from mmseg.ops import DepthwiseSeparableConvModule
+from ..builder import HEADS
+from .fcn_head import FCNHead
+
+
+@HEADS.register_module()
+class DepthwiseSeparableFCNHead(FCNHead):
+    """Depthwise-Separable Fully Convolutional Network for Semantic
+    Segmentation.
+
+    This head is implemented according to Fast-SCNN.
+    Args:
+        in_channels(int): Number of output channels of FFM.
+
+        channels(int): Number of middle-stage channels in the decode head.
+
+        concat_input(bool): Whether to concatenate original decode input into
+            the result of several consecutive convolution layers.
+
+        num_classes(int): Used to determine the dimension of
+            final prediction tensor.
+
+        in_index(int): Correspond with 'out_indices' in FastSCNN backbone.
+
+        norm_cfg (dict|None): Config of norm layers.
+
+        align_corners (bool): align_corners argument of F.interpolate.
+
+        loss_decode(dict): Config of loss type and some
+            relevant additional options.
+    """
```

Comment:
No newline is needed between arguments.
Default value is missing.

Reply:
New lines fixed.
There is no default value specified in the code. Is it necessary to modify the code?